Data center endpoints v2 (#1575)
saucelabs/sauce-docs: docs/basics/data-center-endpoints.md

import TabItem from '@theme/TabItem';

The data center you use with Sauce Labs determines:
  * The location of the servers where your tests are run
  * Where related artifacts are stored

>   * Your ability to run headless tests, meaning tests run on a browser that does not have a user interface

Reviewer: ```suggestion
  * Your ability to run headless tests, which are tests run on a browser that do not have a user interface
```
